src/socketio/namespace.py

```diff
@@ -13,13 +13,28 @@ def on_disconnect(self):
 
     async def on_init_route(self, data):
         print("init route", data)
-        ROUTE = data
+        ROUTE.append(data)
 
-    async def on_add_point_ack(self, data):
-        print(data)
+    async def on_add_point_route_ack(self, data):
+        print("add point route", data)
+        ROUTE.append(data)
```

Comment:
`add point ack` returns a whole list so we also need to replace all of route with the new route